tutorials/others/ubuntu/tutorial.yml

```
@@ -0,0 +1,26 @@
+id: 78a3be56-5d51-4ec3-8629-0dd27c352ab5
+
+builder: Canonical
```

```suggestion
project_id: e5a16b6f-763c-4cc9-a2a2-7d3ae70babe9
```
Builders have been renamed to "Projects" and instead of identifying them by name in tutorial.yml, we now use their UUID (this is brand new, I was waiting for the implementation to be completed before telling you ^^).